.github/workflows/linux.yml

```diff
@@ -223,10 +221,11 @@ jobs:
       - name: Download OpenVINO package
         uses: actions/download-artifact@v4
         with:
-          name: openvino_package
+          name: ${{ env.OV_ARTIFACT_NAME }}
           path: ${{ env.INSTALL_DIR }}
```

Comment:
BTW, I would treat OpenVINO provider as artifacts downloader as well.
E.g. 
```
- uses: openvinotoolkit/openvino/.github/actions/openvino_provider@master
    commit_build_key: 'ubuntu20_x86_64'
    revision: 'latest_available'
    path: ${{ env.OPENVINO_DIR }}
```
and later:
```
  run: |
    source ${{ env.OPENVINO_DIR }}/ setupvar.sh
```

w/o extra job to call current version of OpenVINO provider

Reply:
It's not possible when OV provider is running in a separate job on its own Azure-based runner, if I understand you correctly. If tokenizers build will also be moved to Azure-based runners, download step will not be required at all